fix-code-3/main.cpp

```diff
@@ -9,10 +9,10 @@ int main() {
   int total_hours_on_assignments = hours_per_assignment * num_assignments;
 
   // Calculate the total number of lines of code written on assignments.
-  int num_assignments = 16;
+  int num_assignments11 = 16;
```

Comment:
Why is this variable named num_assignments11? Why not just reuse the existing num_assignments variable?